app/commands/v2/import.rb

```diff
@@ -1,27 +1,24 @@
 module Commands
   module V2
-    class Import
-      def initialize(params)
-        @params = params
-      end
+    class Import < BaseCommand
```

Comment:
This might give a bit of weird results by inheriting from BaseCommand, BaseCommand wraps that `call` in a transaction, which will probably interfere with the transaction in this class, and the send_downstream needs to happen outside of that transaction, it also creates a Event (which you might want to do). but I didn't inherit from BaseCommand because of those hidden unexpected side effects.